### PR TITLE
Fix #141904 xviagrahot.com

### LINKED ISSUE #141904: xviagrahot.com
### Issue URL (Ads)
 [https://xviagrahot.com/webspor-ile-canli-mac-izleme-keyfi.html](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fxviagrahot.com%2Fwebspor-ile-canli-mac-izleme-keyfi.html)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Browser extension v4.1.53
Browser: | Chrome
Stealth mode: | disabled
Filters: | <b>Ad Blocking:</b><br/>AdGuard Base<br/><br/><b>Privacy:</b><br/>AdGuard URL Tracking, <br/>AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/>AdGuard Social Media<br/><br/><b>Annoyances:</b><br/>AdGuard Annoyances<br/><br/><b>Language-specific:</b><br/>AdGuard Turkish

### PR DESCRIPTION
**Pull request by gork7777**
### To avoid invalid pull requests, please check and confirm following terms

- [x] This is not an ad/bug report;
- [x] My code follows the [guidelines](https://github.com/AdguardTeam/AdguardFilters/blob/master/CONTRIBUTING.md) and [syntax](https://kb.adguard.com/general/how-to-create-your-own-ad-filters) of this project;
- [x] I have performed a self-review of my own changes;
- [x] My changes do not break web sites, apps and files structure.

### If the problem does not fall under any category that is listed here, please write a comment below in corresponding section

- [x] Missed ads or ad leftovers;
- [ ] Website or app doesn't work properly;
- [ ] AdGuard gets detected on a website;
- [ ] Missed analytics or tracker;
- [ ] Social media buttons — share, like, tweet, etc;
- [ ] Annoyances — pop-ups, cookie warnings, etc;
- [ ] Filters maintenance.

### Enter the issue address

https://github.com/AdguardTeam/AdguardFilters/issues/141904

### Terms

- [x] By submitting this issue, I agree that pull request does not contain private info and all conditions are met

### CODE REVIEW

File: TurkishFilter/sections/specific.txt [hunk 1]
```diff
@@ -431,7 +431,6 @@ habermotto.com##div[id^="kanews_widget_ads-"]
 memurlar.net##a[rel="sponsored"]
 maksatbilgi.com###tie-block_2320.media-overlay
 maksatbilgi.com##.theiaStickySidebar > .stream-item-widget
-||zf00.cloud/popmyshake.js
```
Comment (Sergey-Lyapin) [inline TurkishFilter/sections/specific.txt:434]: Hi @gork7777 
Why did you remove this rule?

Comment (gork7777) [reply TurkishFilter/sections/specific.txt:434]: @Sergey-Lyapin Doesn't exist anymore. At least I cannot reproduce again: https://github.com/AdguardTeam/AdguardFilters/commit/9a89de4016bf12ac590d2e3e594020cfab06adb1

Comment (Sergey-Lyapin) [reply TurkishFilter/sections/specific.txt:434]: @gork7777 
But the script is still openable, it can be used somewhere else, it's better to keep the rule.

**Review by Sergey-Lyapin — APPROVED** [review]
